sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalog.scala

```diff
@@ -621,20 +697,136 @@ class CosmosCatalog
     options.asCaseSensitiveMap().asScala.toMap
   }
 
+  // scalastyle:off cyclomatic.complexity
+  // scalastyle:off method.length
+  private def generateTblProperties
+  (
+    metadata: (CosmosContainerProperties, List[FeedRange], Option[(ThroughputProperties, Boolean)])
+  ): util.HashMap[String, String] = {
+
+    val containerProperties: CosmosContainerProperties = metadata._1
+    val feedRanges: List[FeedRange] = metadata._2
+    val throughputPropertiesOption: Option[(ThroughputProperties, Boolean)] = metadata._3
+
+    val indexingPolicySnapshotJson =  Option.apply(containerProperties.getIndexingPolicy) match {
+      case Some(p) => ModelBridgeInternal.getJsonSerializable(p).toJson
+      case None => "null"
+    }
+
+    val changeFeedPolicyPolicySnapshotJson = Option.apply(containerProperties.getChangeFeedPolicy) match {
+      case Some(p) => ModelBridgeInternal.getJsonSerializable(p).toJson
```

Comment:
This is currently in preview, maybe we should not expose it? It might change before GA, so what's the need?

Reply:
Fair point - removed it for now